src/components/v2/Table/TableContainer/TableContianer.types.tsx

```diff
@@ -0,0 +1,13 @@
+import { DropdownItemProps } from '../../../../'
```

Comment:
Would you mind importing the `DropdownItemProps` from the DropdownItem component directory?
```suggestion
import { DropdownItemProps } from '../../../../'
```